design/images/node_reuse_flow.svg

```diff
@@ -0,0 +1 @@
27164,0,0)" xlink:href="#bd"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,11.351234567901235,0)" xlink:href="#W"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,20.116666666666667,0)" xlink:href="#aR"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,33.22098765432099,0)" xlink:href="#X"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,41.98641975308642,0)" xlink:href="#ad"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,49.87530864197531,0)" xlink:href="#W"/></g><g id="M"><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,0,0)" xlink:href="#Y"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,4.382716049382716,0)" xlink:href="#T"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,13.148148148148149,0)" xlink:href="#W"/></g><g id="N"><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,0,0)" xlink:href="#aa"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,3.462345679012346,0)" xlink:href="#R"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,12.227777777777778,0)" xlink:href="#ai"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,20.99320987654321,0)" xlink:href="#W"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,29.758641975308645,0)" xlink:href="#aa"/></g><path d="M238-179c0 84-86 86-175 82L45 0H11l48-248c82 0 179-12 179 69zm-34 1c0-53-64-42-116-43l-19 98c63 0 135 9 135-55" id="be"/><g id="O"><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,0,0)" xlink:href="#be"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,10.518518518518519,0)" xlink:href="#Z"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,15.733950617283952,0)" xlink:href="#X"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,24.499382716049386,0)" xlink:href="#ad"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,32.38827160493827,0)" xlink:href="#U"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,35.85061728395061,0)" xlink:href="#ae"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,43.7395061728395,0)" xlink:href="#U"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,47.20185185185184,0)" xlink:href="#X"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,55.967283950617265,0)" xlink:href="#V"/></g><g id="P"><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,0,0)" xlink:href="#T"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,8.765432098765432,0)" xlink:href="#X"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,17.530864197530864,0)" xlink:href="#ae"/><use transform="matrix(0.043827160493827164,0,0,0.043827160493827164,25.419753086419753,0)" xlink:href="#Y"/></g></defs></g></svg>
```

Comment:
It'd probably be better to put the SVG into a separate file?

Reply:
Hi @hardys, thanks for the review! Not fully sure, what you meant by putting the SVG into a separate file. Should we move _node_reuse_flow.svg_  from design/images to another folder or we better make it .png to have it under images?